src/backend/bridge/dml/executor/plan_executor.cpp

```diff
@@ -35,15 +35,124 @@ executor::ExecutorContext *BuildExecutorContext(ParamListInfoData *param_list,
 /*
  * Added for network invoking efficiently
  */
-executor::ExecutorContext *BuildExecutorContext(const std::vector<Value> &params,
-                                                concurrency::Transaction *txn);
+executor::ExecutorContext *BuildExecutorContext(
+    const std::vector<Value> &params, concurrency::Transaction *txn);
 
 executor::AbstractExecutor *BuildExecutorTree(
     executor::AbstractExecutor *root, const planner::AbstractPlan *plan,
     executor::ExecutorContext *executor_context);
 
 void CleanExecutorTree(executor::AbstractExecutor *root);
 
+/*
+ * Execute the subplan and get a value from the result.
+ */
+Value PlanExecutor::ExecutePlanGetValue(const planner::AbstractPlan *plan,
+                                        const std::vector<Value> &params) {
+  peloton_status p_status;
+
+  //  if (plan == nullptr) return p_status;
```

Comment:
Do we need this check  or not ? If not, please remove it and add a comment saying how we will handle empty plans.